spotbugs-exclude.xml

```diff
@@ -698,6 +698,45 @@
             <And>
                 <Class name="io.undertow.servlet.core.SessionListenerBridge"/>
                 <Method name="sessionDestroyed"/>
+            </And>
+              <And>
+                <!-- TODO: figure out lambda ref? -->
```

Comment:
This comment seems to be unrelated to throws runnable rule

Reply:
AFAIR lambdas trigger those rules.